rust/src/smb/smb.rs

```diff
@@ -80,6 +81,16 @@ pub static mut SMB_CFG_MAX_READ_QUEUE_CNT: u32 = 64;
 pub static mut SMB_CFG_MAX_WRITE_SIZE: u32 = 16777216;
 pub static mut SMB_CFG_MAX_WRITE_QUEUE_SIZE: u32 = 67108864;
 pub static mut SMB_CFG_MAX_WRITE_QUEUE_CNT: u32 = 64;
+/// max size of the per state guid2name cache
+pub static mut SMB_CFG_MAX_GUID_CACHE_SIZE: usize = 1024;
```

Comment:
side note, but it would be good to have something like `lazy_static` to say, yes this is a global variable, but once initialization is done, it is a constant and it can be read safely...

Reply:
Can be done as a future optimization.